### DISCUSSION TITLE
Pushover Script

### DISCUSSION THREAD

**Discussion by oelidoc** [camueller/SmartApplianceEnabler#536] [Installation und Betrieb]
Hallo,

erst einmal vielen Dank für das tolle Programm an den Autor Axel Müller.

Nur eine Kleinigkeit: Ich würde gern Pushover für die  Benachrichtigungen verwenden. Meine Fähigkeiten zur Erstellung eines Scriptes gehen jedoch gegen Null. Vielleicht hat ja jemand erfolgreich Pushover am Start und wäre bereit, mir ein entsprechendes Script zur Verfügung zu stellen.

Vielen Dank im voraus

oelidoc

**Comment by camueller** [comment]
Ich selbst nutzt Telegram für Benachrichtigungen und andere User haben Scrips für Signal und Prowl erstellt. Wenn Du also Pushover nutzt, kannst Du das als Gelegenheit sehen, endlich etwas über Scripting zu lernen. Leider vergeben die meisten Menschen solche Chancen und hoffen lieber, dass andere sich darum kümmern. So ein Script für für SAE zu erstellen ist wirklich nicht schwer!

**Reply by oelidoc** [reply]
Ich nutze für SAE bisher Signal, habe aber für FHEM Pushover im Einsatz und wollte das lediglich etwas vereinheitlichen.
Ich bin knapp 60 Jahre alt und arbeite 50 bis 60 Stunden die Woche an und mit Menschen. Ich habe mir in meiner Freizeit als Anwender viel an Hausautomation und jetzt auch Photovoltaik beigebracht. Aber mir fehlen für Sachen wie Scripte schlichtweg jegliche Voraussetzungen. Ich habe eine schulische und akademische Ausbildung in Abwesenheit jeglicher Form von EDV durchlaufen. Programmiersprachen sind für mich Fremdsprachen, mit denen ich nie etwas zu tun gehabt habe. Und nein, ich erwarte nicht, dass sich andere um mich kümmern und mir fertigen Code abliefern. Ich versuche alles immer erst mal alleine hinzubekommen. Aber meine Ressourcen sind begrenzt und "endlich etwas über Scripting zu lernen" sprengt derzeit sicher meinen zeitlichen und inhaltlichen Rahmen. Dann bleibe ich halt bei Signal. Kein Problem. Wollte ja nur mal fragen.
Gruß
oelidoc
